docs/configure-vars-env.md

```diff
@@ -0,0 +1,36 @@
+# Configure my environment
+
+## Copy environment variables
+
+A `.env` file should be created in the repository root, to keep configuration default on the dev setup:
+
+```
+cp example.env .env
```

Comment:
Maybe clarify that this is not needed if someone ran the bootstrap.sh script for the setup.

Reply:
Maybe just describe what the env file is for here and move the copy and path details to  [docs/manual-setup.md](https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/files#diff-8cdfbac6f7d4b921d1c3bfa25820fc82aff6dcbd0aad86e17d9ad557ec9871ab)

Reply:
I fixed it here : https://github.com/juliushaertl/nextcloud-docker-dev/pull/87/commits/9b3ed6ee3c9b3b4030df6361dbe9a42b12484310